include/aspect/boundary_traction/interface.h

```diff
@@ -125,7 +125,193 @@ namespace aspect
          */
         const GeometryModel::Interface<dim> *geometry_model;
     };
+    template <int dim>
+    class Manager : public ::aspect::SimulatorAccess<dim>
+    {
+      public:
+        /**
+         * Destructor. Made virtual since this class has virtual member
+         * functions.
+         */
+        ~Manager () override;
+
+        /**
+         * A function that is called at the beginning of each time step and
+         * calls the corresponding functions of all created plugins.
+         *
+         * The point of this function is to allow complex boundary velocity
```

Comment:
You still reference the velocity in many places of the header and the source file. The simplest way to fix this is to search for velocity and replace it with traction in all places you find it (you can use ctrl+f in VS code to do this quite quick).

```suggestion
         * The point of this function is to allow complex boundary traction
```